saskatoon/fixtures/member-organization.json

```diff
@@ -8,8 +8,7 @@
     "civil_name": "Les Fruits Défendus",
     "description": "Les Fruits Défendus is Montreal’s volunteer-led urban fruit harvesting collective hosted by Santropol Roulant. The initiative connects local fruit tree owners with volunteer harvesters and helps ensure that this valuable local food resource does not go to waste.",
     "phone": "514-360-8566",
-    "contact_person": 1,
-    "contact_person_role": "Fake person",
+    "contact_person": [1],
```

Comment:
Does the change to the fixture is necessary? It's supposed to be taken care of automatically while loading the fixture ?

Reply:
hello @tristanlatr 
I have rebased the branch.
I get deserialization error when I loaddata and contact_person is an int and not a list.

traceback:
```sh
raise base.DeserializationError.WithData(e.original_exc, d['model'], d.get('pk'), e.pk)
django.core.serializers.base.DeserializationError: Problem installing fixture '/mnt/d/niabc/Programming/Outreachy/Contributions/saskatoon-ng/saskatoon/fixtures/member-organization.json': 'int' object is not iterable: (member.organization:pk=2) field_value was '1'
```

Reply:
I'm rebasing again. You guys are merging FAST😂

Reply:
Personally, I gave up about interactive rebasing. When there are lots of changes a merge commit can be more efficient. But it’s largely a aesthetic choice :)